designcompose/src/main/java/com/android/designcompose/DocServer.kt

```diff
@@ -530,15 +537,14 @@ internal fun DocServer.doc(
     return null
 }
 
-internal fun DocServer.branches(docId: String): HashMap<String, String>? {
-    val mediaDocs: HashMap<String, String> =
+internal fun DocServer.branches(docId: FigmaDocId): HashMap<FigmaDocId, String>? {
+    val mediaDocs: HashMap<FigmaDocId, String> =
         hashMapOf(
-            "7rvM6aVWe0jZCm7jhO9ITx" to "Media 1",
-            "S3n4mhNgoHzNxCCHhmrVcR" to "Media 2",
-            "5n0LhOQ6wOiDxrH0YUVhJS" to "Media 4",
-            "dui99iAKZ273s7RN11Z9Ak" to "Media 5",
-            "2DQtQOf6U26mA8dqBie3gT" to "Media Nova",
+            FigmaDocId("7rvM6aVWe0jZCm7jhO9ITx") to "Media 1",
+            FigmaDocId("S3n4mhNgoHzNxCCHhmrVcR") to "Media 2",
+            FigmaDocId("5n0LhOQ6wOiDxrH0YUVhJS") to "Media 4",
+            FigmaDocId("dui99iAKZ273s7RN11Z9Ak") to "Media 5",
+            FigmaDocId("2DQtQOf6U26mA8dqBie3gT") to "Media Nova",
         )
     return mediaDocs
```

Comment:
This... never should have been checked in. But this should be fixed in a seperate PR.